### ISSUE TITLE
Easier Setup

### ISSUE THREAD

**Issue by peq** [wurstscript/wurst4vscode#11]
Problems / possible improvements:
- When no folder is opened there should be a warning or it should automatically find the right root folder.
- when starting with a wrong java version it will give a strange error message currently (http://oi63.tinypic.com/2j5kzfb.jpg)
- the compiler (jar files) should be downloaded automatically
- the dependency file / basic project structure should be created automatically

**Comment by Cokemonkey11** [comment]
It seems that the readme also doesn't mention anything about finding the extension on vscode package manager.

**Comment by peq** [comment]
Right, that's because the readme is the same shown in the package manager ;D

Probably not a good idea.

**Comment by Cokemonkey11** [comment]
RustyCode for example points to itself when you open it in package manager:

https://github.com/saviorisdead/RustyCode
